tests/e2e/cluster_destroy_test.go

```diff
@@ -13,7 +13,7 @@ var _ = Describe("TF Test", func() {
 
 				// Generate/build cluster by profile selected
 				profile := CI.LoadProfileYamlFileByENV()
-				err := CI.DestroyRHCSClusterByProfile(token, profile)
+				err := CI.DestroyRHCSClusterByProfile(GetToken(), profile)
```

Comment:
may I know why we rename it to _test?

Reply:
because golang does not understand the file without the `test` postfix, it is still working without it,
but it gives warnings.